verify-docker-versions

```diff
@@ -24,10 +24,19 @@ compare_versions() {
   return 0
 }
 
+# test that user has rights to use docker
+
+DOCKER_ENGINE_INFO_STDERR=$(docker info 2>&1 >/dev/null)
+if [[ -n "$(echo $DOCKER_ENGINE_INFO_STDERR | grep 'permission denied')" ]]; then
+  >&2 echo "Local user has insufficient permissions to connect to the docker daemon."
+  >&2 echo "Please add user to 'docker' group to be able to access it or check the documentation for your host OS."
```

Comment:
I would add here that if you were just added to the group, you need to log out and in again. This is a fairly common problem for first-installers.

Reply:
Good suggestion. Added